src/anm/graphics/text.js

```diff
@@ -171,7 +171,7 @@ Text.prototype.bounds = function() {
  * @param {Number} pt.y
  * @return {Boolean} is point inside
  */
-Text.prototype.inside = function(pt) {
+Text.prototype.contains = function(pt) {
```

Comment:
https://github.com/Animatron/player/pull/384/files#diff-a072a28291a97d7ac72e402357d42fc6R1979

It is, though the fact it's always named `contains` probably requires `Text`, `Video` etc. to also have `inBounds` method, just to make it similar everywhere.